torch/nn/modules/conv.py

```diff
@@ -10,7 +10,8 @@
 class _ConvNd(Module):
 
     def __init__(self, in_channels, out_channels, kernel_size, stride,
-                 padding, dilation, transposed, output_padding, groups, bias):
+                 padding, dilation, transposed, output_padding, groups, bias,
+                 initializer=dict()):
```

Comment:
It's not good to have default values of [functions to be mutable elements](http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/). Might be better to set it to `None` and create the dict inside the function
```python
def fun(initializer=None):
    if initializer is None:
        initializer = dict()
```